benchmarks/inference-server/vllm/secret-templates/secret-provider.tftpl

```diff
@@ -0,0 +1,21 @@
+%{ for hugging_face_token_secret in hugging_face_token_secret_list ~}
+
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
```

Comment:
Thoughts on us refactoring this to the inference-server/ level?

It would be nice to standardize this across all of the inference serving frameworks rather than copy paste.

Reply:
Done